swagger2.0/oic.r.rule.actioncollection.swagger.json

```diff
@@ -0,0 +1,218 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OCF Rule Action Collection",
+    "version": "20190910",
+    "license": {
+      "name": "OCF Data Model License",
+      "url": "https://github.com/openconnectivityfoundation/core/blob/e28a9e0a92e17042ba3e83661e4c0fbce8bdc4ba/LICENSE.md",
+      "x-copyright": "copyright 2019 Open Connectivity Foundation, Inc. All rights reserved."
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/RuleActionCollectionResURI?if=oic.if.ll": {
+      "get": {
+        "description": "A Collection of Links to one or more Rule Actions, which are processed when the Rule Expression evaluates to \"true\".",
+        "parameters": [
+          {"$ref": "#/parameters/interface-all"}
+        ],
+        "responses": {
+          "200": {
+            "description" : "Retrieves the rule action Links.",
+            "x-example":
+            [
+              {
+                "href": "/myruleaction",
+                "rt":   ["oic.r.rule.action"],
+                "if":   ["oic.if.rw","oic.if.baseline"],
+                "p":    {"bm": 3},
+                "eps": [
+                    {"ep": "coaps://[fe80::b1d6]:1111"}
+                ]
+              }
+            ],
+            "schema": { "$ref": "#/definitions/slinklist" }
+          }
+        }
+      }
+    },
+    "/RuleActionCollectionResURI?if=oic.if.baseline": {
+      "get": {
+        "description": "A Collection of Links to one or more Rule Actions, which are processed when the Rule Expression evaluates to \"true\".",
+        "parameters": [
+          {"$ref": "#/parameters/interface-all"}
+        ],
+        "responses": {
+          "200": {
+            "description" : "Retrieves the rule action Links.",
+            "x-example":
+            {
+              "rt":  ["oic.r.rule.actioncollection"],
+              "if":  ["oic.if.ll","oic.if.baseline"],
```

Comment:
Add "oic.if.create"?

Reply:
No.  Not in the rule action collection.

Reply:
From CR 1968:

The Collection of Rule Actions  may be empty. A Client may add Rule Actions to the Collection using the create ("oic.if.create") OCF Interface.

So it seems as though we either need to add support for oic.if.create in this schema or change the spec text in the CR so that they are aligned.